### PR TITLE
JDK-8267574: Dead code in HtmlStyle/HtmlDocletWriter

### PR DESCRIPTION
**Pull request by jonathan-gibbons**
Please review a simple `noreg-cleanup` change to remove some dead code.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8267574](https://bugs.openjdk.java.net/browse/JDK-8267574): Dead code in HtmlStyle/HtmlDocletWriter

### Reviewers
 * [Hannes Wallnöfer](https://openjdk.java.net/census#hannesw) (@hns - **Reviewer**) ⚠️ Review applies to c33a8bd97f487d547cd7da1ea7f988f112424a9c

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/4154/head:pull/4154` \
`$ git checkout pull/4154`

Update a local copy of the PR: \
`$ git checkout pull/4154` \
`$ git pull https://git.openjdk.java.net/jdk pull/4154/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 4154`

View PR using the GUI difftool: \
`$ git pr show -t 4154`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/4154.diff">https://git.openjdk.java.net/jdk/pull/4154.diff</a>

</details>

### CODE REVIEW

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css [hunk 1]
```diff
@@ -577,8 +577,7 @@ h1.hidden {
     margin-right:10px;
     display:inline-block;
 }
-div.block div.deprecation-comment, div.block div.block span.emphasized-phrase,
-div.block div.block span.interface-name {
+div.block div.deprecation-comment, div.block div.block span.emphasized-phrase {
```
Comment (hns) [inline src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css:580]: A quick search through the code and generated docs suggests that `emphasized-phrase` is unused as well. Should this also be removed?

**Review by hns — APPROVED** [review]
Looks good, but I found another potential cleanup target close by (see inline comment).

File: src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css (hunk 1 above)
Comment (jonathan-gibbons) [reply src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css:580]: Let me investigate a test program  to compare the class names in the style sheet and the members of HtmlStyle, and then decide whether to merge that here or do a separate commit/PR.

Comment (jonathan-gibbons) [reply src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/resources/stylesheet.css]: Test program reveals differences that need to be investigated. 
See https://bugs.openjdk.java.net/browse/JDK-8267709

For now, I've just removed `emphasized-phrase`

**Pull request closed without merging**